src/Functions.Worker.Extensions.OpenAI.Kusto/Functions.Worker.Extensions.OpenAI.Kusto.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <Description>Microsoft Azure Functions .NET Isolated Worker Extension for OpenAI Kusto</Description>
+    <WebJobsVersion Condition="'$(WebJobsVersion)' == ''">$(VersionPrefix)-$(VersionSuffix)</WebJobsVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Functions.Worker.Extensions.Abstractions" Version="1.3.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <AssemblyAttribute Include="Microsoft.Azure.Functions.Worker.Extensions.Abstractions.ExtensionInformationAttribute">
+      <_Parameter1>Microsoft.Azure.WebJobs.Extensions.OpenAI.Kusto</_Parameter1>
+      <_Parameter2>$(WebJobsVersion)</_Parameter2>
```

Comment:
we have separate versioning for Kusto and AI Search package now, it should match with respective Kusto package

Reply:
I added two new properties to Directory.build.props with the version numbers directly, since we'd want to consolidate the changes so that we don't have to update webjobs and worker to always be in sync. We'd only have to update Directory.build.props to update the version. Let me know if this looks ok.